### PR TITLE
Ballistics : add ACE magazines in vanilla magwells

### PR DESCRIPTION
**Pull request by Laid3acK**
**When merged this pull request will:**
- update some ACE magazines in vanilla magazineWell as https://github.com/acemod/ACE3/pull/7562 and remove useless CBA magazineWell in the CfgWeapons.

### CODE REVIEW

**Comment by Laid3acK** [comment]
Really proper like that in my opinion. Thanks commy2. :)

File: addons/ballistics/CfgWeapons.hpp [hunk 1]
```diff
@@ -11,19 +11,18 @@ class CfgWeapons {
         ACE_barrelLength = 730;
         ACE_barrelTwist = 381.0;
         initSpeed = -1.0;
-        magazineWell[] += {
-            "CBA_50BMG_M107"
```
Comment (dedmen) [inline addons/ballistics/CfgWeapons.hpp]: why remove a valid magazine well? This one should be added on CBA side I'd say

Comment (Laid3acK) [reply addons/ballistics/CfgWeapons.hpp]: The both work in-game. As you want.
Maybe the name of this CBA magwell should be changed,
it's not really appropriate to the vanilla game and this rifle.

**Comment by Laid3acK** [comment]
I just saw that the Lynx has his 12.7x108 magwell in the AiO config,
i just add it in the cfgweapons (probably a BI forgetting).

**Comment by commy2** [comment]
>not a forgetting, the Lynx can't have a magwell with the vanilla game.

Why would that be?

**Comment by Laid3acK** [comment]
A weapon configured with two magazines without the same `initSpeed` included in the `magazines[]` and also the `magazineWell[]` could work correctly with the vanilla game ?
I forgot that the Lynx has an `initSpeed=0`.  So probably yes.

**Comment by Mike-MF** [comment]
Closing due to age and/or inactivity, unlikely to be finished or has to be reworked due to other changes over the years. May be re-opened if someone finds it useful, even better just open a new up-to-date PR.

**Pull request closed without merging**